D1673/P1673.bs

```diff
@@ -2,13 +2,13 @@
 Title: A free function linear algebra interface based on the BLAS
 Shortname: P1673
 Status: iso/P
-Level: 7
+Level: 8
```

Comment:
is that the revision number?  did you mean to update in this PR?

Reply:
@dalg24 Yes, though I realize now I probably should have left this alone and dealt with it later.  (I just wanted the list of changes to match with the version number.)